src/cs/usfca/edu/histfavcheckout/model/TopUserResponse.java

```diff
@@ -0,0 +1,65 @@
+package cs.usfca.edu.histfavcheckout.model;
+
+import java.io.Serializable;
+
+public class TopUserResponse implements Serializable {
+	private String userName;
+	private String email;
+	private long favsCount;
+	private long checkoutsCount;
+	private long ratingsCount;
+	
+	public TopUserResponse() {}
+	
+	public TopUserResponse(String userName, String email, long favsCount, long checkoutsCount, long ratingsCount) {
+		this.userName = userName;
+		this.email = email;
+		this.favsCount = favsCount;
+		this.checkoutsCount = checkoutsCount;
+		this.ratingsCount = ratingsCount;
+	}
+
+	public String getUserName() {
+		return userName;
+	}
+
+	public void setUserName(String userName) {
```

Comment:
I don't really care about this, but looks like we aren't using most of the setters here, as values are set in the constructor. Do you think we should keep them? Also same in TopUser & UserInfoResponse classes.

Reply:
I think it is good to remove them and safe to keep them. For now, I am not changing the implementation since there are lot of models that has unused getters and setters. If we decide to address the cleanup we can clean this up along with it.